jdaviz/configs/imviz/tests/test_viewers.py

```diff
@@ -97,6 +97,15 @@ def test_mastviz_config():
     assert im.app.data_collection[0].shape == (2, 2)
 
 
+def test_zoom_center_radius_init(imviz_helper):
+    """Regression test for https://github.com/spacetelescope/jdaviz/issues/3217"""
+    arr = np.ones((10, 10))
+    imviz_helper.load_data(arr, data_label='my_array')
+    assert imviz_helper.default_viewer._obj.state.zoom_center_x > 0
+    assert imviz_helper.default_viewer._obj.state.zoom_center_y > 0
+    assert imviz_helper.default_viewer._obj.state.zoom_radius > 0

```

Comment:
Can we be a little more specific? Maybe `allclose` check instead of just non-zero check?

Reply:
I intentionally left this flexible so if the zoom behavior changes this does not break except for the regression (where the values were left uninitialized at their default values).  Other tests handle specific zoom-tool behavior already.  But if you'd still prefer to enforce the current values, I can easily make that change.